crates/storage-rocksdb/src/lib.rs

```diff
@@ -64,6 +65,16 @@ const FSM_TABLE_NAME: &str = "fsm";
 const TIMERS_TABLE_NAME: &str = "timers";
 const JOURNAL_TABLE_NAME: &str = "journal";
 
+type StorageFormatVersionType = u32;
+
+/// Version of the used storage format. Whenever you introduce a breaking change, this version needs
+/// to be incremented in order to distinguish the new from the old versions. Ideally, you also add
+/// a migration path from the old to the new format.
+const STORAGE_FORMAT_VERSION: StorageFormatVersionType = 0;
```

Comment:
Tip: Starting with version 1 means that we no hidden default initialization will cause us to believe that we are on the correct version.

Reply:
This is true. Will change it to `1`.